### PR TITLE
Added custom fail reporter so build does not fail with warnings

### LINKED ISSUE #213: Warnings are being promoted to errors
We're getting closer :smile: 

screen shot 2015-02-10 at 15 33 37

Possibly:-
- Warnings (e.g. above) shouldn't break builds

Nicer solution?
- Maybe a `--strict` mode where all warnings become errors

### PR DESCRIPTION
**Pull request by AlbertoElias**
Fixes #213

### CODE REVIEW

File: lib/tasks/verify.js [hunk 1]
```diff
@@ -60,17 +62,37 @@ function pathsToGlob(paths) {
...
 
 /**
+ * Custom fail reporter that doesn't fail on Scss Lint warnings
+ */
+function failReporter() {
+	var stream = through.obj(function(file, enc, cb) {
+		var error;
+		if (file.scsslint.errors > 0) {
+			error = new gutil.PluginError('gulp-scss-lint', {
+				message: 'ScssLint failed for: ' + file.relative,
```
Comment (kaelig) [inline lib/tasks/verify.js]: SCSS-Lint

File: lib/tasks/verify.js [hunk 2]
```diff
@@ -60,17 +62,37 @@ function pathsToGlob(paths) {
 }
 
 /**
+ * Custom fail reporter that doesn't fail on Scss Lint warnings
```
Comment (kaelig) [inline lib/tasks/verify.js]: SCSS-Lint

**Comment by kaelig** [comment]
GG :+1:

File: lib/tasks/verify.js (hunk 1 above)
Comment (AlbertoElias) [inline lib/tasks/verify.js]: Actually, that line of code is copied from the fail reporter gulp-scsslint offers. But I will update anyway

**Pull request merged**